CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# v1.3.7
```

Comment:
```suggestion
# v1.4.0
```

Reply:
Yep I'm making another PR at the moment that fixes that and document the Yjs DevTools changes.